ils_middleware/dags/alma.py

```diff
@@ -0,0 +1,182 @@
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.models import Variable
+from airflow.operators.dummy import DummyOperator
```

Comment:
I think `DummyOperator` is depreciated, the replacement is [EmptyOperator](https://airflow.apache.org/docs/apache-airflow/stable/_api/airflow/operators/empty/index.html), I believe you just replace this line with `from airflow.operators.empty import EmptyOperator` and then replace the `DummyOperator` with `EmptyOperator` in the DAG.